tckdb/backend/app/tests/conversions/test_converter.py

```diff
@@ -562,7 +562,11 @@ def test_inchi_key_from_inchi():
 def test_adjlist_from_smiles():
     """Test getting an adjacency list from a SMILES descriptor"""
     for adjlist, smiles in zip(test_set['adjlist'], test_set['smiles']):
-        assert adjlist == converter.adjlist_from_smiles(smiles)
+        adjlist_2 = converter.adjlist_from_smiles(smiles)
+        mol1, mol2 = Molecule().from_adjacency_list(adjlist), Molecule().from_adjacency_list(adjlist_2)
+        assert mol1.is_isomorphic(mol2) == True
```

Comment:
I think we don't need the `== True` at the end

Reply:
Yeah, I could drop the `== True` but I think it reads nicely in this instance